weldx/utility.py

```diff
@@ -423,6 +435,14 @@ def xr_interp_like(
     else:  # assume da2 to be dict-like
         sel_coords = da2
 
+    # store and strip pint units at this point, since the unit is lost during
+    # interpolation and because of some other conflicts. Unit is restored before
+    # returning the result.
```

Comment:
I will do some testing if pint integration is improved with xarray 0.16.0 in this case